src/vm/functions/mod.rs

```diff
@@ -139,11 +148,27 @@ fn native_eq(args: &[Value]) -> Result<Value> {
         Ok(Value::Bool(true))
     } else {
         let first = &args[0];
+        // Using `fold` rather than `all` to prevent short-circuiting. 
         let result = args.iter().fold(true, |acc, x| acc && (*x == *first));
         Ok(Value::Bool(result))
     }
 }
 
+fn native_isnull(args: &[Value]) -> Result<Value> {
+    // TODO: see note in `native_eq` above ListTypes equality...
+    if args.len() == 0 {
+        // TODO: Should no input args be allowed, if so should it return true or false?
```

Comment:
This should be a syntax error.  `isnull?` should take at least one argument.

Reply:
Why do we need `isnull?`.  Isn't `(eq? variable_name 'null)` sufficient?  Seems unnecessary to me.

Reply:
Intention is to reduce the repetitive verbose code in most smart contracts.
We return `'null` / `void` in several common native functions -- `get`, `fetch-contract-entry`, `fetch-entry`, `get-block-info` seem to often need a null check afterwards. 

Although, `isnull?` alone doesn't help very much. I'm tempted to suggest we add an if-null control flow function.

Reply:
This disinclines me to add `isnull?` as a native function.  Generally speaking, if it can be done in a subroutine or library, it should be done that way.

Reply:
Removed `isnull?` -- in favor of a future function that can abort on null.